Disable building txt documentation. This is a temporary workaround,
as I have found an Ubuntu 10.10 system which throws errors during
building this that I'd like to ultimately fix. The error manifests
itself from the end of LinuxDocTools.pm with the following messages
during do_install:

| - Building txt docs
| Processing file ./guide
| troff: fatal error: can't find macro file s
|  fmt_txt::postASP: Empty output file, error when calling groff. Aborting...

Upstream-Status: Inappropriate [Other]
Temporary workaround which disables documentation.

Signed-off-by: Scott Garman <scott.a.garman@intel.com>

diff -urN linuxdoc-tools-0.9.66.orig//doc/Makedoc.sh linuxdoc-tools-0.9.66/doc/Makedoc.sh
--- linuxdoc-tools-0.9.66.orig//doc/Makedoc.sh	2009-11-09 11:58:25.000000000 -0800
+++ linuxdoc-tools-0.9.66/doc/Makedoc.sh	2011-03-04 17:37:24.788923998 -0800
@@ -40,11 +40,11 @@
 
 chmod u+x $TMPDIR/linuxdoc
 
-if [ -n "`which groff`" ]; then
-	ln -s $TMPDIR/linuxdoc $TMPDIR/sgml2txt
-	echo "- Building txt docs" >&2
-	$TMPDIR/sgml2txt -b 1 ./guide
-fi
+#if [ -n "`which groff`" ]; then
+#	ln -s $TMPDIR/linuxdoc $TMPDIR/sgml2txt
+#	echo "- Building txt docs" >&2
+#	$TMPDIR/sgml2txt -b 1 ./guide
+#fi
 
 if [ -n "`which latex`" ]; then
 	ln -s $TMPDIR/linuxdoc $TMPDIR/sgml2latex
